src/main/resources/changelog.xml

```diff
@@ -1112,6 +1112,28 @@
 			baseColumnNames="label_id"/>
 	</changeSet>
 
+	<changeSet id="addedNotificationClasses" author="Arjan Seijs and Capser Henkes">
+		<createTable tableName="notification">
+			<column name="id" type="bigint" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false" />
+			</column>
+			<column name="message" type="varchar(255)"></column>
+			<column name="event" type="varchar(255)"></column>
+			<column name="link" type="varchar(255)"></column>
+			<column name="user_id" type="bigint"></column>
+			<column name="date" type="datetime"></column>
+		</createTable>
+		<createTable tableName="notifications_to_users">
+			<column name="user_id" type="bigint"></column>
+			<column name="notification_id" type="bigint"></column>
+			<column name="isRead" type="boolean"></column>
+		</createTable>
+		<!--<addPrimaryKey tableName="notification_to_users" columnNames="user_id, notification_id"/>-->
```

Comment:
I commented these constrains out because they didn't work yet, but we should fix them. Also, edit the changelog so that the newly added changelog is appended at the bottom (below Vera's changeset). This because the changelog will always be applied in-order.